packages/fast-usdc/src/types.ts

```diff
@@ -55,4 +55,21 @@ export interface PoolMetrics extends PoolStats {
   shareWorth: Ratio;
 }
 
+export interface ChainPolicy {
+  nobleContractAddress: EvmHash;
+  cctpTokenMessengerAddress: EvmHash;
+  confirmations: number;
+  chainName: string;
+  bridgeContractAddress?: EvmHash;
+  eventFilter?: string;
```

Comment:
I _think_ this is the same for all chains, so we can maybe move this up to `FeedPolicy`